lib/video_cutter/video_cutter.ex

```diff
@@ -0,0 +1,81 @@
+defmodule Membrane.VideoCutter do
+  use Membrane.Filter
+
+  alias Membrane.Caps.Video.Raw
+
+  def_options intervals: [
+                spec: [{pos_integer(), pos_integer() | :infinity}],
```

Comment:
0 is not a `pos_integer`
```suggestion
                spec: [{non_neg_integer(), pos_integer() | :infinity}],
```

EDIT: You may use `Membrane.Time.t()` like mentioned in comment below